backend/src/consts.ts

```diff
@@ -8,3 +8,28 @@ const COFFEE_CHAT_BINGO_BOARD = [
 export default COFFEE_CHAT_BINGO_BOARD;
 
 export const DISABLE_DELETE_ALL_COFFEE_CHATS = true;
+
+export const ALL_ROLES: Role[] = [
```

Comment:
This is duplicated with `frontend/src/consts.ts` unfortunately :(. But we don't currently have a way of sharing constants between frontend and backend (`constants.js` isn't actually used on the backend)